### ISSUE TITLE
[4.6] Diogenes not starting on Debian Sid

### ISSUE THREAD

**Issue by Doc73** [pjheslin/diogenes#89]
Here is the output:
```
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
server stdout: 
Starting Diogenes...

server stdout: <h1>Software error:</h1>
<pre>Can't locate JSON.pm in @INC (you may need to install the JSON module) (@INC contains: /usr/local/diogenes/server /usr/local/diogenes/server/../dependencies/CPAN /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.34.0 /usr/local/share/perl/5.34.0 /usr/lib/x86_64-linux-gnu/perl5/5.34 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.34 /usr/share/perl/5.34 /usr/local/lib/site_perl) at /usr/local/diogenes/server/Diogenes/Script.pm line 35.
BEGIN failed--compilation aborted at /usr/local/diogenes/server/Diogenes/Script.pm line 35.
Compilation failed in require at /usr/local/diogenes/server/diogenes-server.pl line 47.
BEGIN failed--compilation aborted at /usr/local/diogenes/server/diogenes-server.pl line 47.
</pre>
<p>
For help, please send mail to this site's webmaster, giving this error message 
and the time and date of the error.

</p>

server stderr: [Tue Sep 20 09:48:54 2022] diogenes-server.pl: Can't locate JSON.pm in @INC (you may need to install the JSON module) (@INC contains: /usr/local/diogenes/server /usr/local/diogenes/server/../dependencies/CPAN /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.34.0 /usr/local/share/perl/5.34.0 /usr/lib/x86_64-linux-gnu/perl5/5.34 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.34 /usr/share/perl/5.34 /usr/local/lib/site_perl) at /usr/local/diogenes/server/Diogenes/Script.pm line 35.
[Tue Sep 20 09:48:54 2022] diogenes-server.pl: BEGIN failed--compilation aborted at /usr/local/diogenes/server/Diogenes/Script.pm line 35.
[Tue Sep 20 09:48:54 2022] diogenes-server.pl: Compilation failed in require at /usr/local/diogenes/server/diogenes-server.pl line 47.
[Tue Sep 20 09:48:54 2022] diogenes-server.pl: BEGIN failed--compilation aborted at /usr/local/diogenes/server/diogenes-server.pl line 47.

Diogenes server exited (or failed to start)
[14189:0920/094859.624318:ERROR:nss_util.cc(349)] After loading Root Certs, loaded==false: NSS error code: -8018
```

Thank you very much in advance,
DC

**Comment by pjheslin** [comment]
Thanks for the prompt report!  This is the same problem as issue #88.  I tested the new version of Diogenes on no fewer than 6 different computers, and all of them must have JSON.pm installed, so this bug was not triggered.  It's a new dependency and I'll have to package a substitute for this module with Diogenes.  I'll fix this oversight ASAP and release a new version.

**Comment by Doc73** [comment]
Wait a moment!
Maybe, you could add as a dependence the `libjson-perl` package, available in official repositories, instead of build this package inside the deb.
I'll do some tests

**Comment by Doc73** [comment]
@pjheslin 
Done, but in Debian Stable!
Without  `libjson-perl` works very well; without it, same error!

Please, don't pack this module in your deb, but add the package as dependency. It pulls down also `libcommon-sense-perl libjson-xs-perl libtypes-serialiser-perl`.

**Comment by pjheslin** [comment]
I've fixed this in version 4.6.1 by adding JSON::Tiny.pm, which weighs in at 7KB, which is a drop in the ocean.  If Diogenes were purely a Linux application, I would create a dependency as you suggest, but to keep things working across Windows/Mac/Linux sometimes requires working in a way that is not optimal for all platforms.

**Comment by Doc73** [comment]
@pjheslin 
Yes, I undertand!
Anyway, I can confirm that the new version 4.6.1 works fine!

Thank you so much!
